src/Uno.UI/UI/Xaml/FrameworkTemplate.cs

```diff
@@ -24,7 +25,8 @@ namespace Windows.UI.Xaml
 	/// Defines a builder to be used in <see cref="FrameworkTemplate"/>
 	/// </summary>
 	public delegate View? FrameworkTemplateBuilder(object? owner);
-	
+
+	[ContentProperty(Name = "Template")]
```

Comment:
There's no `Template` property, where did you find this one?

Reply:
It may be an artifact of the way the `FrameworkTemplate` class is defined in C++. Thing is that this property is not visible in the C# projection, so I don't think we should be showing it either.

Reply:
@jeromelaban Why wouldn't this have effect on user-defined classes that inherit `FrameworkTemplate`?

---

> Thing is that this property is not visible in the C# projection

I'm not sure what you mean, I see it in VS, do you mean something else?

![image](https://user-images.githubusercontent.com/31348972/211564135-d57455ed-36cb-4a3e-a476-a771456e4732.png)

Reply:
I mean that the `Template` property does not exist, even if the `ContentProperty` refers to it.

Reply:
Yes, I see the property doesn't exist in `FrameworkTemplate`

What I mean is:

```csharp
// user-code
public class MyClass : FrameworkTemplate
{
    // define a property named "Template"
    // Even though MyClass doesn't have ContentProperty attribute, it is inherited from FrameworkTemplate.
}
```

Can't this happen?

Reply:
I see. This is a scenario that is not supported by user code, mainly because there's no public constructor available on WinUI.

Reply:
@jeromelaban Why does it have to be public? Isn't a protected constructor sufficient? https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.frameworktemplate.-ctor?view=windows-app-sdk-1.2

Reply:
Ah! This may have changed from UWP then :) But anyways, it does not change the fact that the property does not exist, but it does not seem to cause issues at this point, so we can keep it as-is.